.circleci/config.yml

```diff
@@ -24,3 +35,69 @@ jobs:
             . venv/bin/activate
             cd api/python && pytest --cov=./
             codecov
+  build:
+    docker:
+      - image: circleci/python:3.6-jessie
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "api/python/setup.py" }}-{{ checksum "Makefile" }}
+      - run:
+          name: install python dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -e api/python[tests]
+      - save_cache:
+          key: v1-dependency-cache-{{ checksum "api/python/setup.py" }}-{{ checksum "Makefile" }}
+          paths:
+            - "venv"
+      - run:
+          name: run tests
+          command: |
+            . venv/bin/activate
+            cd api/python && pytest --cov=./
+            codecov
+      - store_artifacts:
+          path: htmlcov/
+
+  deploy:
+    docker:
+      - image: circleci/python:3.6-jessie
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "api/python/setup.py" }}-{{ checksum "Makefile" }}
+      - run:
+          name: install python dependencies
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -e api/python[tests]
+      - save_cache:
+          key: v1-dependency-cache-{{ checksum "api/python/setup.py" }}-{{ checksum "Makefile" }}
+          paths:
+            - "venv"
+      - run:
+          name: verify git tag vs. version
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            python api/python/setup.py verify
+      - run:
+          name: init .pypirc
+          command: |
+            echo -e "[pypi]" >> ~/.pypirc
+            echo -e "username = quiltdata" >> ~/.pypirc
+            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
```

Comment:
are we sure this doesn't end up in the logs? a safer way of doing this might be `envsubst` on a template file.

Reply:
Do all this logs is:

```
#!/bin/bash -eo pipefail
echo -e "[pypi]" >> ~/.pypirc
echo -e "username = quiltdata" >> ~/.pypirc
echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
```

I don't think `envsubst` is any safer.